tools/plugins/network/src/main/kotlin/net/corda/cli/plugins/network/OnboardMgm.kt

```diff
@@ -109,9 +113,8 @@ class OnboardMgm : Runnable, BaseOnboard() {
     }
 
     private val cpi by lazy {
-        val parametersCpiFile = cpiFile
-        if(parametersCpiFile != null) {
-            return@lazy parametersCpiFile
+        if (cpiFile != null) {
+            return@lazy cpiFile
```

Comment:
```suggestion
            return@lazy cpiFile!!
```
This will remove the need for the not-null assertion below